tests/payload_tools/fitness.py

```
@@ -3,14 +3,24 @@ def build_create_workout_payload(
     description: str = "A very intense bicep workout",
     instructions: str = "Lift the dumbbells",
     workout_type: str = "rep",
-    equipment: list = []
+    equipment: list = [],
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)